Vagrantfile

```diff
@@ -5,11 +5,11 @@
 #
 # SKIP_BCC_BUILD: Set to skip the building bcc from source
 
-$ubuntu_18_deps = <<EOF
+$ubuntu_deps = <<EOF
 apt-get -qq update
 apt-get -qq install linux-headers-$(uname -r) binutils-dev
 apt-get -qq install bison cmake flex g++ git libelf-dev zlib1g-dev libfl-dev systemtap-sdt-dev
```

Comment:
Does this work on xenial? #1284 does a bit of effort to get gcc 7 all set up

Reply:
Ah crap it probably does. What a day well spent yesterday.

Reply:
Hmm well the other builds are using g++ so maybe it'd be better to stick to g++ for all releases for now.